src/server/saved_objects/service/create_saved_objects_service.js

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-import { SavedObjectsRepository, ScopedSavedObjectsClientProvider } from './lib';
+import { SavedObjectsRepository, ScopedSavedObjectsClientProvider, SavedObjectsRepositoryProvider } from './lib';
```

Comment:
Is `SavedObjectsRepository` used anymore?

Reply:
Not directly, but it's being exported as part of the Saved Object Service's [public interface](https://github.com/legrego/kibana/blob/1dc221bed33b9acc32d40dd90a0ccb1813a41053/src/server/saved_objects/service/create_saved_objects_service.js#L85), which I didn't change as part of this PR.

Reply:
Ah, right!